specification/ai/HealthInsights/HealthInsights.RadiologyInsights/client.tsp

```diff
@@ -100,3 +100,15 @@ interface RadiologyInsightsClient {
   "ClinicalDocumentContent",
   "csharp"
```

Comment:
Add java to the scope of existing renames instead of adding a new `clientName` entry.

```suggestion
  "csharp,java"
```

Reply:
adjusted. Thanks for the advice!